eslint.config.mjs

```diff
@@ -50,14 +58,23 @@ export default [
     rules: {
       ...ts.configs['eslint-recommended'].rules,
       ...ts.configs.recommended.rules,
-      'no-undef': 'off' // redundant - TS will fail to compile with undefined vars
+      'no-undef': 'off', // redundant - TS will fail to compile with undefined vars
+      '@typescript-eslint/no-unused-vars': [
+        'error',
+        { ignoreRestSiblings: true, argsIgnorePattern: '^_', destructuredArrayIgnorePattern: '^_' }
+      ],
+      '@typescript-eslint/no-empty-function': 'off', // allow noop functions
```

Comment:
do we have a lot of functions that require this carve out? would prefer to keep it on if possible

Reply:
Current totals in this branch at time of typing:
Rule on: 254 problems (173 errors, 81 warnings)
Rule off: 248 problems (167 errors, 81 warnings)

So 6 instances. I would rather keep it on too, could have a single exemption in a resources file and any file where we're using an empty func imports that?  I'm also trying to reduce refactoring before the omni release.
e.g. `import { noop } from 'resources/utils`

Reply:
We can still make the above change and disable it completely (with one exemption as mentioned above) but as an intermediary I restricted the rule to arrow functions only.